src/main/java/persistence/entity/EntityMangerImpl.java

```diff
@@ -0,0 +1,84 @@
+package persistence.entity;
+
+import jakarta.persistence.Id;
+import jdbc.JdbcTemplate;
+import jdbc.RowMapper;
+import persistence.Person;
+import persistence.sql.ddl.exception.IdAnnotationMissingException;
+import persistence.sql.dialect.H2Dialect;
+import persistence.sql.dml.*;
+import persistence.sql.mapping.ColumnData;
+import persistence.sql.mapping.Columns;
+
+import java.lang.reflect.Field;
+import java.util.Arrays;
+
+public class EntityMangerImpl implements EntityManger {
+    private JdbcTemplate jdbcTemplate;
+    private H2Dialect dialect;
```

Comment:
```suggestion
public class EntityMangerImpl implements EntityManger {
    private final JdbcTemplate jdbcTemplate;
    private final H2Dialect dialect;
```
필드를 불변(final) 로 가져간다면 어떤 이점이 있을까요? 😃

Reply:
좀 찾아봤는데 GC 성능에 대한 내용도 덕분에 읽어보게 됐네요 감사합니다~